Client/reasn-client/apps/web/app/Nav.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styles from './Nav.module.css';
+
+
+type Link = {
+    label: string
+    href: string
+};
+
+const Nav = () => {
+return(
+<body>
+  <header>
+    <nav
+      className="
+        flex flex-wrap
+        items-center
+        justify-between
+        w-full
+        h-20
+        text-lg text-white
+        bg-black
+        rounded-b-2xl
+      "
+    >
+    <div>
+        <a className="block text-white font-bold text-3xl mx-60">Reasn.</a>
+    </div>
+
+    <div className="flex space-x-4 items-center mx-60"> 
+      <a className="block text-white mr-8">logowanie</a>   
+      <a className="px-8 py-1.5 text-white border-2 border-white rounded-full">rejestracja</a> 
```

Comment:
```suggestion
      <a className="px-6 py-1.5 border-2 border-white rounded-2xl">rejestracja</a> 
```
Same as above - it inherits the text-white, and imo - it looks better with smaller padding and smaller rounding, what do you think?

Reply:
Same as above changed and I agree with you so also changed padding and rounding.